plugins/inputs/sqlserver/sqlqueriesV1.go

```diff
@@ -558,7 +558,7 @@ SELECT database_name, num_of_writes_persec
 FROM #baselinewritten
 WHERE datafile_type = ''ROWS''
 ) as V
-PIVOT(SUM(num_of_writes_persec) FOR database_name IN (' + @ColumnName + ')) AS PVTTabl
+PIVOT(SUM(num_of_writes_persec) FOR database_name IN (' + @ColumnName + ')) AS PVTTable
```

Comment:
We should not be modifying queries used here

```suggestion
PIVOT(SUM(num_of_writes_persec) FOR database_name IN (' + @ColumnName + ')) AS PVTTabl
```

Reply:
Done.